### DISCUSSION TITLE
Option to change the ND between the old and new one

### DISCUSSION THREAD

**Discussion by ButterLander2222** [magknight/787aviatorsTracker#839] [Ideas]
Currently the new ND has no "plan" mode. This is an important feature that now is unavailable. It was available on the old ND. This means the old ND has one big advantage over the new one. However, the old ND was also very low quality and is hard use. Since old and new both have their uses (old during flight planning, new in-flight), there could be an option to switch between them until the new one has a plan mode.

**Comment by aeroniemi** [comment]
this won't be added - the old ND is functionally obsolete and adding it would require just as much effort as adding plan mode to the new one
